lte/gateway/c/oai/common/log.c

```diff
@@ -563,7 +563,8 @@ static char* log_get_readable_cur_time(time_t* cur_time) {
   // get the current local time
   *cur_time = time(NULL);
   // get the current local time in readable string format
-  return (strtok(ctime(cur_time), "\n"));
+  char buf[26];
```

Comment:
This is exact buf required, I would gone with larger buffer than required.

Reply:
@pshelar missed this comment before merging this PR. Could you please suggest an alternative size for the buffer? I can fix it in a separate PR.

Reply:
32 bytes.